server/core/runtime/policy/conftest_executor.go

```diff
@@ -127,6 +127,10 @@ func (c *ConfTestExecutor) Run(_ context.Context, prjCtx command.ProjectContext,
 	return output, policyErr
 }
 
+func sanitizeName(name string) string {
```

Comment:
Why do we need to do this? Probably best to also comment the rason.

Reply:
It's not needed, removed.